features/contain.yml

```diff
@@ -0,0 +1,14 @@
+name: Containment
+description: The `contain` CSS property TODO.
```

Comment:
```suggestion
description: The `contain` CSS property marks an element and its contents as being independent from the rest of the document. This permits the browser to avoid slower layout calculations.
```

Reply:
I removed this file when splitting into the different types of containment.